### ISSUE TITLE
createMuiTheme was deprecated

### ISSUE THREAD

**Issue by RobertHeim** [siriwatknp/mui-treasury#1148]
Hey and thanks for your work on the great library!

I see the following error on runs, builds and every test:
```
  console.error
    Material-UI: the createMuiTheme function was renamed to createTheme.
    
    You should use `import { createTheme } from '@material-ui/core/styles'`

      at createMuiTheme (node_modules/@material-ui/core/styles/createTheme.js:114:15)
      at Object.<anonymous> (node_modules/@mui-treasury/layout/components/Root/Root.js:20:44)
      at Object.<anonymous> (node_modules/@mui-treasury/layout/components/Root/index.js:13:36)
```

I am using `@mui-treasury/layout 4.5.0`

It would be great if you could release a fix for this.

**Comment by siriwatknp** [comment]
Are you using `@material-ui/core` version 4.x.x?

**Comment by barthicus** [comment]
> Are you using `@material-ui/core` version 4.x.x?

@siriwatknp  I could answer for @RobertHeim 
This happens on `4.12.0` and up because `createMuiTheme` is now depracated. Haven't tried on v5 though. The result will be the same I think.

From the Material UI release notes: https://github.com/mui-org/material-ui/releases/tag/v4.12.0

> [theme] Deprecate createMuiTheme (#26004) @m4theushw

I think you should revert to one version back which is `4.11.4`. At least for now.

**Comment by siriwatknp** [comment]
@barthicus thanks for the info. @RobertHeim I will release a patch that use `createTheme` this weekend (for material-ui v4). However, encourage to migrate to `@mui-treasury/layout v5` and `@material-ui/core v5`

**Comment by RobertHeim** [comment]
Hey, yes, we are using 4.12.2 since v5 is currently beta, but we will consider to mirgate to v5 anyway. Thanks for the upcoming patch!

**Comment by cstephe** [comment]
So happy to hear this,  I am writing the base of our app and plan to give to junior devs but this error was something I didn't want junior devs to see straight away on a clean project for them.   Thanks for patching it, excited for the v5 updates as well once they are out of beta,

**Comment by siriwatknp** [comment]
Fixed with `@mui-treasury/layout v4.5.1`. https://github.com/siriwatknp/mui-treasury/tree/%40mui-treasury/layout%404.5.1
